keel-redis/src/main/kotlin/com/netflix/spinnaker/keel/redis/RedisResourceRepository.kt

```diff
@@ -58,30 +59,30 @@ class RedisResourceRepository(
     }
 
   override fun store(resource: Resource<*>) {
-    redisClient.withCommandsClient<Long?> { redis: JedisCommands ->
+    redisClient.withCommandsClient<Long> { redis: JedisCommands ->
```

Comment:
Is it even worth keeping redis around if we think sql is a better fit for the data? Or, are we going to maintain and test both to fit with our story of "you can still run on redis we just don't"

Reply:
That was my assumption but I was asking myself the same question, yeah.

Reply:
Hmm. Let's keep redis for now, and support both ways. I think it makes sense to have an easy redis story as well as a recommended sql story.

Reply:
Yeah, I think I'd be tempted to abandon it if it was time consuming to maintain, but it's not so far.

Reply:
We want to keep redis as an option for now.